src/util/safelywritablefile.cpp

```diff
@@ -263,6 +241,51 @@ bool SafelyWritableFile::commit() {
     return true;
 }
 
+void SafelyWritableFile::useTempFileWithPrefix() {
+    QString origFilePath = QFileInfo(m_origFileName).canonicalPath();
+    QString origFileCompleteBasename = QFileInfo(m_origFileName).completeBaseName();
+    QString origFileSuffix = QFileInfo(m_origFileName).suffix();
+    QString tempFileCompleteBasename = kSafelyWritableTempFilePrefix +
+            origFileCompleteBasename + '.' + origFileSuffix;
```

Comment:
There is no need to slit off the suffox

Reply:
completeBaseName() returned without the suffix, I had to add the suffix in order to not hit the format checks that will come later on, or did i compose the the temp file name wrong?